src/drivers/uavcannode/uavcannode_params.c

```diff
@@ -50,3 +50,83 @@ PARAM_DEFINE_INT32(CANNODE_NODE_ID, 120);
  * @group UAVCAN
  */
 PARAM_DEFINE_INT32(CANNODE_BITRATE, 1000000);
+
+/**
+ * UAVCAN barometer publication period
+ * @unit us
+ * @min 0
+ * @max 1000000
+ * @group UAVCAN
+ */
+PARAM_DEFINE_INT32(UAVCAN_BARO_T, 100000);
```

Comment:
@JacobCrabill FYI the `sensors` module is able to downsample (average) data (SENS_BARO_RATE, SENS_MAG_RATE), we could start bringing it into the CAN node use case. I already intend to do this for things like optical_flow where it's desirably to closely synchronize integrated flow + gyro prior to going out over CAN.

Reply:
I intended for that variable to limit the publication rate only as described for the `uavcan.pubp-X` variables at https://legacy.uavcan.org/Specification/6._Application_level_functions/#standard-configuration-parameters. You could limit the sensor rate too if you don't need the higher rate on the node.